src/global.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module '*.scss';
+declare module '*.css';
```

Comment:
What does this file do?

Reply:
It makes css/sccss files allowed in typescript. I suppose scss isn't needed rn so can remove that line.

https://stackoverflow.com/a/51536973